python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1696,6 +1697,66 @@ def test_parquet_writer_chunked_partitioned(tmpdir_factory, return_meta):
     assert_eq(got_pd, got_cudf)
 
 
+@pytest.mark.parametrize(
+    "max_file_size,max_file_size_in_bytes",
+    [("500KB", 500000), ("1000KB", 1000000)],
```

Comment:
Just to cover the size parsing code
```suggestion
    [("500KB", 500000), ("MB", 1000000)],
```